src/pkg/backup/backup.go

```diff
@@ -143,7 +165,7 @@ func New(
 			SkippedNotFound:           notFound,
 			SkippedInvalidOneNoteFile: invalidONFile,
 		},
-	}
+	}, nil
```

Comment:
The backup struct is a bit large to be part of a return value in a multi-value return.  Can you swap it to something like this, please?

```
bup := Backup{...}

return &bup, nil
```